dotnet-build.sh

```diff
@@ -65,12 +91,19 @@ echo "building $CS_PROJECT_FILE"
 dotnet build -c $RELEASE --no-restore $CS_PROJECT_FILE
 
 # run tests
-echo "running tests"
-for f in *.Test/*.csproj
-do
-  echo "Processing $f file..."
-  dotnet test "$f" --no-restore -c $RELEASE -r "$DIST/test-results"  
-done
+if [ $WITH_TESTING ]
+then
+  echo "running tests"
+  for f in *.Test/*.csproj
+  do
+    echo "Processing $f file..."
+    dotnet test "$f" -c $RELEASE -r "$DIST/test-results" --filter "TestCategory!=[Integration]" /p:CollectCoverage=true /p:CoverletOutputFormat="opencover" /p:CoverletOutput="$DIST/test-coverlet"
```

Comment:
Why are you applying a filter? There is no Integration category in our default test framework :stuck_out_tongue:

Reply:
Not yet indeed, but will be :-)

Reply:
Hope not btw:-)

Reply:
Test categories make more sense when you start using **real** databases to test with.
You can spin up a dozen environments (with dozens of different versions of databases) and run your specific category tests there (f.e to test differences between postgre/mysql).

There are some interesting tests we can do that way but as of now, we don't really have a need for it.